### PR TITLE
Suggested edits to data

### PR DESCRIPTION
**Pull request by DmitrySharabin**
Hello there! I used Mavo to suggest the following edits:
I added mv-uploads
Preview my changes here: https://mavo.io/docs/?storage=https%3A%2F%2Fraw.githubusercontent.com%2FDmitrySharabin%2Fmavo.io%2Fmaster%2Fdocs%2Findex.json&docs-storage=https%3A%2F%2Fgithub.com%2Fmavoweb%2Fmavo.io%2Fdocs%2Findex.json

### CODE REVIEW

File: docs/index.json [hunk 1]
```diff
@@ -202,6 +202,14 @@
 			"on": "Collections",
 			"purpose": "How many items should the collection start with? If not provided, it is 1, unless `mv-like` is also specified, in which case it's 0 (to prevent infinite recursion)."
 		},
+		{
+			"name": "mv-uploads",
+			"on": "Mavo root",
+			"purpose": "Provide different upload backend than storage backend.\n**Note:** For now, it is only available in [the development version](https://mavo.io/get/).",
+			"live": "yes",
```
Comment (LeaVerou) [inline docs/index.json:209]: Is it actually live? Like, if you set it with an expression does it update?

Comment (DmitrySharabin) [reply docs/index.json:209]: Yep! I made a [simple pen](https://codepen.io/dmitrysharabin/pen/LYpEBeZ) to prove it. 😊Please, have a look.

Comment (LeaVerou) [reply docs/index.json:209]: Nice!

**Comment by LeaVerou** [comment]
Merged, thanks!

**Pull request merged**